bin/diagnostic.js

```diff
@@ -1 +1,15 @@
 'use strict'
+const ask = require('../lib/ask.js')
+
+let answer = 'not empty'
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ")
+  if (answer < 1 || answer > 10) {
+   console.log("the guess needs to be between 1 and 10!")
+   break
+  }
+  else {
+    console.log("You guessed it!")
+  }
+}
```

Comment:
You were on the right track here Chris. When you include the `break` line in your `if` statement it breaks out of the loop completely. So if someone guesses a number less than 1 or greater than 10 it tells you to guess between 1 and 10 but then breaks out of the loop so you can't keep guessing.

You also haven't given the `else` statement the correct number, so if you guess anything between 1 and 10 it tells you that you are right. You can take a look at the solution branch but also reach out if you want to go over it in more detail.